geometry_msgs/PoseStamped[] demo
bool use_loss_func
---

